spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -301,12 +311,6 @@ def test_output_from_c_libraries(kernel, capsys):
 libc.printf(('Hello from C\\n').encode('utf8'))
 """
 
-    # Without Wurlitzer there's not output generated
-    # by the kernel
-    reply = kernel.do_execute(code, True)
-    captured = capsys.readouterr()
-    assert captured.out == ''
```

Comment:
Why did you remove this? Are you loading `wurlitzer` unconditionally now?

Reply:
I had problems with wurlitzer and I was loading it to avoid exceptions, but I fixed the problems in wurlitzer, so I am not sure if this part is still a problem.

Reply:
from this line (not something I added): https://github.com/spyder-ide/spyder-kernels/blob/5496e4596dabda05d8583e2533fcdb14ebde2c9c/spyder_kernels/console/start.py#L126
I would expect wurlitzer to be loaded inconditionnaly?